packages/cfd/src/Helpers/cfd-config.ts

```diff
@@ -0,0 +1,59 @@
+export const CFD_PLATFORMS = {
+    MT5: 'mt5',
+    DXTRADE: 'dxtrade',
+    DERIVEZ: 'derivez',
+    CTRADER: 'ctrader',
+    CFD: 'cfd',
+    CFDS: 'CFDs',
+} as const;
+
+export const CATEGORY = {
+    DEMO: 'demo',
+    REAL: 'real',
+} as const;
+
+export const MARKET_TYPE = {
+    SYNTHETIC: 'synthetic',
+    GAMING: 'gaming',
+    FINANCIAL: 'financial',
+    ALL: 'all',
+} as const;
+
+export const MOBILE_PLATFORMS = {
+    IOS: 'ios',
+    HAUWEI: 'huawei',
+    ANDROID: 'android',
+} as const;
+
+export const OS_PLATFORMS = {
```

Comment:
`MOBILE_PLATFORMS` followed by `DESKTOP_PLATFORMS` ?

Reply:
MACOS is desktop or more possibly OS?
@shontzu-deriv

Reply:
Renamed